### DISCUSSION TITLE
Still developed?

### DISCUSSION THREAD

**Discussion by SyndicatedPillbug** [davish/obsidian-full-calendar#573] [Polls]
Wondering if there's still any active development on this neat plugin? I don't really want to go through the trouble of setting it up if not, and from the bugs it looks like there are more and more issues arising without anyone checking on them.

**Comment by QuAzI** [comment]
As there no github activity from @davish for months it looks like this nice calendar absolutely abandoned.

**Reply by SyndicatedPillbug** [reply]
Initiate sobbing.

**Comment by Hiperon** [comment]
Thank you for asking. I was wondering about this.

**Comment by SyndicatedPillbug** [comment]
I ended up subscribing to the Morgen app because I really need multi-calendar and task integration. They say that they have an Obsidian plugin on the way, so that could be cool - though it's a shame there isn't an open-source option atm.

**Reply by wheeleran** [reply]
it's out and in the repository

**Comment by chrsdk** [comment]
I am still using this plugin as I find it very useful and what I'm doing with it so far hasn't caused any issues yet. But yeah, what a shame that it seems abandoned. Not only this plugin, but also calendar and periodic notes.

**Comment by YouFoundJK** [comment]
Do check my [fork](https://github.com/YouFoundJK/plugin-full-calendar) out! Most of the issues raised here are now fixed and most  features (Google Cal integration, Recurring tasks, ...) have also been added.

**Reply by wheeleran** [reply]
thanks for all your great work, really. is it correct that the remote calDAV are still read-only?

**Reply by YouFoundJK** [reply]
That's right. Among remote calendar only Google Calendar has read and write capability. As long as the calDAV provider doesn't require additional APIs it should be fairly easy to implement. Feel free to open a request in my fork. 

I have some plans on migrating from the backend `dav` library that this plugin currently uses to something that is currently maintained (still figuring it out). So will take some time before I get to that.

UPDATE (21 Nov '25): a good working calDAV isn't as easy to implement as I had thought. Tried `tsdav` library but that doesnt seems to work out for some reason. So I still have to figure it out.

**Reply by chrsdk** [reply]
Will this be in the community plugin repository?

**Reply by YouFoundJK** [reply]
@chrsdk I hope so... I set up [request](https://github.com/obsidianmd/obsidian-releases/pull/7249) quite a long back but haven't got response yet.

**Reply by YouFoundJK** [reply]
@wheeleran Now calDAV should be two-syncing [here](https://github.com/YouFoundJK/plugin-full-calendar/releases/latest).

**Reply by YouFoundJK** [reply]
Now available in the [community store](https://community.obsidian.md/plugins/full-calendar-remastered).
